numba/cuda/compiler.py

```diff
@@ -404,13 +394,15 @@ def __init__(self, name, llvmir, options):
         self.cache = {}
         self._extra_options = options.copy()
 
-    def get(self):
+    def get(self, cc=None):
```

Comment:
I've reviewed this out of order, below is a remark about adding a deprecation notice for CC.

Reply:
Deprecation notices /warnings now added